django/thunderstore/api/urls.py

```diff
@@ -147,9 +184,41 @@
         TeamMemberAddAPIView.as_view(),
         name="cyberstorm.team.member.add",
     ),
+    path(
+        "team/<str:team_name>/member/<str:username>/remove/",
+        TeamMemberRemoveAPIView.as_view(),
+        name="cyberstorm.team.member.remove",
+    ),
     path(
         "team/<str:team_id>/service-account/",
         TeamServiceAccountListAPIView.as_view(),
         name="cyberstorm.team.service-account",
     ),
+    path(
+        "team/<str:team_name>/service-account/create/",
+        CreateServiceAccountAPIView.as_view(),
+        name="cyberstorm.team.service-account.create",
+    ),
+    path(
+        "team/<str:team_name>/service-account/delete/<uuid:uuid>/",
+        DeleteServiceAccountAPIView.as_view(),
+        name="cyberstorm.team.service-account.delete",
+    ),
+    path(
+        "user/delete/",
+        DeleteUserAPIView.as_view(),
+        name="cyberstorm.user.delete",
+    ),
+    path(
+        "user/linked-account/<str:provider>/disconnect/",
+        DisconnectUserLinkedAccountAPIView.as_view(),
+        name="cyberstorm.user.linked-account.disconnect",
+    ),
+    path(
+        "team/<str:team_name>/member/<str:team_member>/update/",
+        UpdateTeamMemberAPIView.as_view(),
+        name="cyberstorm.team.member.update",
+    ),
```

Comment:
These also strike me as something that should be under the `current-user` API path as that's what we're operating on.

Alternatively we could use some magic value for referring to the current user and parametrize the API, e.g. `@me` like Discord does.